src/main/java/frc/robot/subsystems/climber/C2025ClimberIO.java

```diff
@@ -0,0 +1,102 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.CANdiConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.NeutralOut;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.CANdi;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.S1StateValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.robot.Robot;
+
+
+public class C2025ClimberIO implements ClimberIO{
+    private final TalonFX motor;
+    private final CANcoder coder;
+    private final CANdi beamBrake1;
+    private final CANdi beamBrake2;
```

Comment:
The CANdi actually has 2 ports for beam brake sensors, so you can just use 1 CANdi